crates/hyperqueue/src/server/client/autoalloc.rs

```diff
@@ -238,6 +242,206 @@ fn create_queue(
     }
 }
 
+struct QueueLimit {
+    walltime: Option<u32>,
+    backlog: Option<u32>,
+    workers: Option<u32>,
+}
+
+// Returns value of given arg, in formats: keyVALUE, key=VALUE, [key, VALUE]
+fn get_arg_by_key(key: String, args: &[String]) -> Option<String> {
```

Comment:
It would be nice to write some Rust unit tests that check how this works for `-qqexp`, `-q qexp`, `-q=qexp` etc.